src/components/libraryoptionseditor/libraryoptionseditor.js

```diff
@@ -215,6 +215,40 @@ function renderSubtitleFetchers(page, availableOptions, libraryOptions) {
     elem.innerHTML = html;
 }
 
+function renderLyricFetchers(page, availableOptions, libraryOptions) {
+    let html = '';
+    const elem = page.querySelector('.lyricFetchers');
+
+    let plugins = availableOptions.LyricFetchers;
+    plugins = getOrderedPlugins(plugins, libraryOptions.LyricFetcherOrder || []);
```

Comment:
This crashes for me because `plugins` is currently undefined. We should add default values for the parameters of `getOrderedPlugins` to prevent that.

```
function getOrderedPlugins(plugins = [], configuredOrder = []) {
```

Reply:
It shouldn't be undefined, are you running against unstable server?
Either way I'm fine with adding default values but if it is still undefined we likely have a larger issue

Reply:
I am running unstable, but not the latest build.

Reply:
I just tested using this branch and latest master and cannot reproduce the issue.

The property was added in https://github.com/jellyfin/jellyfin/pull/12397
My response from `/Libraries/AvailableOptions` is:

Without Plugin:
```json
{
    "MetadataSavers": [
        {
            "Name": "Nfo",
            "DefaultEnabled": true
        }
    ],
    "MetadataReaders": [
        {
            "Name": "Nfo",
            "DefaultEnabled": true
        }
    ],
    "SubtitleFetchers": [],
    "LyricFetchers": [],
    "TypeOptions": [
        {
            "Type": "MusicArtist",
            "MetadataFetchers": [
                {
                    "Name": "TheAudioDB",
                    "DefaultEnabled": false
                },
                {
                    "Name": "MusicBrainz",
                    "DefaultEnabled": true
                }
            ],
            "ImageFetchers": [
                {
                    "Name": "TheAudioDB",
                    "DefaultEnabled": true
                }
            ],
            "SupportedImageTypes": [
                "Primary",
                "Logo",
                "Banner",
                "Backdrop"
            ],
            "DefaultImageOptions": [
                {
                    "Type": "Backdrop",
                    "Limit": 1,
                    "MinWidth": 1280
                },
                {
                    "Type": "Banner",
                    "Limit": 0,
                    "MinWidth": 0
                },
                {
                    "Type": "Art",
                    "Limit": 0,
                    "MinWidth": 0
                },
                {
                    "Type": "Logo",
                    "Limit": 1,
                    "MinWidth": 0
                }
            ]
        },
        {
            "Type": "MusicAlbum",
            "MetadataFetchers": [
                {
                    "Name": "MusicBrainz",
                    "DefaultEnabled": true
                },
                {
                    "Name": "TheAudioDB",
                    "DefaultEnabled": false
                }
            ],
            "ImageFetchers": [
                {
                    "Name": "TheAudioDB",
                    "DefaultEnabled": true
                }
            ],
            "SupportedImageTypes": [
                "Primary",
                "Disc"
            ],
            "DefaultImageOptions": [
                {
                    "Type": "Backdrop",
                    "Limit": 0,
                    "MinWidth": 1280
                },
                {
                    "Type": "Disc",
                    "Limit": 0,
                    "MinWidth": 0
                }
            ]
        },
        {
            "Type": "Audio",
            "MetadataFetchers": [],
            "ImageFetchers": [
                {
                    "Name": "Image Extractor",
                    "DefaultEnabled": true
                }
            ],
            "SupportedImageTypes": [
                "Primary"
            ],
            "DefaultImageOptions": []
        },
        {
            "Type": "MusicVideo",
            "MetadataFetchers": [],
            "ImageFetchers": [
                {
                    "Name": "Embedded Image Extractor",
                    "DefaultEnabled": true
                },
                {
                    "Name": "Screen Grabber",
                    "DefaultEnabled": true
                }
            ],
            "SupportedImageTypes": [
                "Primary",
                "Backdrop",
                "Logo"
            ],
            "DefaultImageOptions": [
                {
                    "Type": "Backdrop",
                    "Limit": 1,
                    "MinWidth": 1280
                },
                {
                    "Type": "Art",
                    "Limit": 0,
                    "MinWidth": 0
                },
                {
                    "Type": "Disc",
                    "Limit": 0,
                    "MinWidth": 0
                },
                {
                    "Type": "Primary",
                    "Limit": 1,
                    "MinWidth": 0
                },
                {
                    "Type": "Banner",
                    "Limit": 0,
                    "MinWidth": 0
                },
                {
                    "Type": "Thumb",
                    "Limit": 1,
                    "MinWidth": 0
                },
                {
                    "Type": "Logo",
                    "Limit": 1,
                    "MinWidth": 0
                }
            ]
        }
    ]
}
```

With Plugin:
```json
{
    "MetadataSavers": [
        {
            "Name": "Nfo",
            "DefaultEnabled": true
        }
    ],
    "MetadataReaders": [
        {
            "Name": "Nfo",
            "DefaultEnabled": true
        }
    ],
    "SubtitleFetchers": [],
    "LyricFetchers": [
        {
            "Name": "LrcLib",
            "DefaultEnabled": true
        }
    ],
    "TypeOptions": [
        {
            "Type": "MusicArtist",
            "MetadataFetchers": [
                {
                    "Name": "TheAudioDB",
                    "DefaultEnabled": false
                },
                {
                    "Name": "MusicBrainz",
                    "DefaultEnabled": true
                }
            ],
            "ImageFetchers": [
                {
                    "Name": "TheAudioDB",
                    "DefaultEnabled": true
                }
            ],
            "SupportedImageTypes": [
                "Primary",
                "Logo",
                "Banner",
                "Backdrop"
            ],
            "DefaultImageOptions": [
                {
                    "Type": "Backdrop",
                    "Limit": 1,
                    "MinWidth": 1280
                },
                {
                    "Type": "Banner",
                    "Limit": 0,
                    "MinWidth": 0
                },
                {
                    "Type": "Art",
                    "Limit": 0,
                    "MinWidth": 0
                },
                {
                    "Type": "Logo",
                    "Limit": 1,
                    "MinWidth": 0
                }
            ]
        },
        {
            "Type": "MusicAlbum",
            "MetadataFetchers": [
                {
                    "Name": "MusicBrainz",
                    "DefaultEnabled": true
                },
                {
                    "Name": "TheAudioDB",
                    "DefaultEnabled": false
                }
            ],
            "ImageFetchers": [
                {
                    "Name": "TheAudioDB",
                    "DefaultEnabled": true
                }
            ],
            "SupportedImageTypes": [
                "Primary",
                "Disc"
            ],
            "DefaultImageOptions": [
                {
                    "Type": "Backdrop",
                    "Limit": 0,
                    "MinWidth": 1280
                },
                {
                    "Type": "Disc",
                    "Limit": 0,
                    "MinWidth": 0
                }
            ]
        },
        {
            "Type": "Audio",
            "MetadataFetchers": [],
            "ImageFetchers": [
                {
                    "Name": "Image Extractor",
                    "DefaultEnabled": true
                }
            ],
            "SupportedImageTypes": [
                "Primary"
            ],
            "DefaultImageOptions": []
        },
        {
            "Type": "MusicVideo",
            "MetadataFetchers": [],
            "ImageFetchers": [
                {
                    "Name": "Embedded Image Extractor",
                    "DefaultEnabled": true
                },
                {
                    "Name": "Screen Grabber",
                    "DefaultEnabled": true
                }
            ],
            "SupportedImageTypes": [
                "Primary",
                "Backdrop",
                "Logo"
            ],
            "DefaultImageOptions": [
                {
                    "Type": "Backdrop",
                    "Limit": 1,
                    "MinWidth": 1280
                },
                {
                    "Type": "Art",
                    "Limit": 0,
                    "MinWidth": 0
                },
                {
                    "Type": "Disc",
                    "Limit": 0,
                    "MinWidth": 0
                },
                {
                    "Type": "Primary",
                    "Limit": 1,
                    "MinWidth": 0
                },
                {
                    "Type": "Banner",
                    "Limit": 0,
                    "MinWidth": 0
                },
                {
                    "Type": "Thumb",
                    "Limit": 1,
                    "MinWidth": 0
                },
                {
                    "Type": "Logo",
                    "Limit": 1,
                    "MinWidth": 0
                }
            ]
        }
    ]
}
```

Reply:
Yeah I'm sure it is because I am running an older unstable build, but we normally _try_ to keep backwards compatibility since web is still bundled in some clients.